src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DatePicker/DatePickerFlyout.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<Page x:Class="SamplesApp.Samples.DatePicker.DatePickerFlyoutSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:ControlTests"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:xamarin="http://nventive.com/xamarin"
+      xmlns:ios="http://nventive.com/ios"
+      xmlns:android="http://nventive.com/android"
+      xmlns:native_ios="using:UIKit"
+      xmlns:native_android="using:Android.Widget"
+      xmlns:controls="using:Uno.UI.Samples.Controls"
+      mc:Ignorable="xamarin android ios native_ios native_android">
+
+	<controls:SampleControl SampleDescription="Sample for Date Picker Flyout. The DatePicker should appear at the bottom on iOS.">
+		<controls:SampleControl.SampleContent>
+			<DataTemplate>
+				<Grid Margin="0,100,0,0">
+					<Grid.RowDefinitions>
+						<RowDefinition Height="Auto" />
+						<RowDefinition Height="Auto" />
+					</Grid.RowDefinitions>
+
+					<Button Content="Click to open a date picker flyout">
```

Comment:
Creating a UI test to press the button and take a screenshot will be important here (even if iOS is not yet tested), this should work on Android.